.circleci/config.yml

```diff
@@ -82,6 +82,9 @@ jobs:
             - src/api/tmp/rubocop_cache_root_dir
             - src/api/tmp/rubocop_cache_rails_dir
       - run: rm -rf src/api/tmp/rubocop*
+      - run:
+          name: Run brakeman
+          command: sudo gem install --no-format-executable brakeman; cd src/api; brakeman --rails6
```

Comment:
I've looked at the brakeman documentation and the option `rails6` is not documented. It runs `brakeman` with some Rails 6 specific checks?

Brakeman can receive the path as an option with `-p / --path`. Doing `cd src/api` is also fine.

Reply:
Shouldn't we include running brakeman as another task/command inside the [`dev:lint` task](https://github.com/openSUSE/open-build-service/blob/24145637ba72265db98f118e424e0bfd2d9e7e5a/src/api/lib/tasks/dev.rake#L63)?

Reply:
> Rails 6 specific checks

```
brakeman --help|grep rails6
    -6, --rails6                     Force Rails 6 mode
```

> Shouldn't we include running brakeman as another task

Can't as it's not there normally, only on CI.